app/Http/Controllers/MessageHistoryController.php

```diff
@@ -29,9 +29,15 @@ public function destroy(MessageHistory $message)
             $this->messageHistoryRepository->delete($message->id);
 
             return redirect(route('history.show', ['history' => $payloadHistoryId]))
-                    ->with('messageSuccess', 'This message history successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This message history successfully deleted'
+                ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/MessageHistoryController.php</br>**Line**: 39</br>Each line in an array declaration must end in a comma